bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/read/TestScanAdapter.java

```diff
@@ -55,17 +57,24 @@
 @RunWith(JUnit4.class)
 public class TestScanAdapter {
 
+  public static final String STOP_KEY = "stopKey";
```

Comment:
All cosmetic changes like static variables should be introduced in a separate PR.

Reply:
@rahulKQL, please create a new PR for the new variables.

Reply:
I would revert these to their previous line, I just refactor these two constants to group with rest of the constant. I added these constant in #2013